prac_09/taxi_test.py

```diff
@@ -0,0 +1,11 @@
+from prac_09.taxi import Taxi
+
+my_taxi = Taxi(name="Prius 1", fuel=100)
+my_taxi.drive(40)
+print(my_taxi)
+print(my_taxi.get_fare())
+
+my_taxi.start_fare()
+my_taxi.drive(100)
```

Comment:
Try adding a print statement for the expected outcome.
If you were unfamiliar with the program, the taxi only driving 60km might seem like an error.